sklearn/linear_model/tests/test_logistic.py

```diff
@@ -254,13 +253,16 @@ def test_elasticnet_l1_ratio_err_helpful(LR):
 # TODO(1.8): remove whole test with deprecation of multi_class
 @pytest.mark.filterwarnings("ignore:.*'multi_class' was deprecated.*:FutureWarning")
 @pytest.mark.parametrize("solver", ["lbfgs", "newton-cg", "sag", "saga"])
-def test_multinomial_binary(solver):
+def test_multinomial_binary(global_random_seed, solver):
```

Comment:
Maybe this test doesn't need global_random_seed as it will be removed, same for `test_multinomial_binary_probabilites`??

Reply:
Remove test now